tests/test_tekton.py

```diff
@@ -905,28 +982,38 @@ def test_any_task_failed_or_cancelled(
     @pytest.mark.parametrize(
         'get_json, expect_results',
         [
-            ({}, []),
-            ({'status': {}}, []),
+            ({}, {}),
+            ({'status': {}}, {}),
             (
                 {"status": {"pipelineResults": [{'name': 'foo', 'value': '1234'}]}},
-                [{'name': 'foo', 'value': '1234'}],
+                {'foo': 1234},
             ),
             (
                 {
                     "status": {
                         "pipelineResults": [
-                            {'name': 'x', 'value': '1'}, {'name': 'y', 'value': '2'},
+                            {'name': 'x', 'value': '{"key": "value"}'}, {'name': 'y', 'value': '2'},
```

Comment:
Could you add a testcase with `{'name': 'x', 'value': 'null'}`? Those are supposed to be excluded from the returned dict

Reply:
done